WcaOnRails/app/models/registration.rb

```diff
@@ -131,6 +131,7 @@ def record_refund(amount, currency_code, stripe_refund_id, stripe_charge_id)
       amount_lowest_denomination: amount * -1,
       currency_code: currency_code,
       stripe_charge_id: stripe_refund_id,
+      original_stripe_charge_id: stripe_charge_id,
```

Comment:
If I understand this correctly, this is a "stripe id". Would it make sense for this to be a "registration_payment id" instead? You'd be able to get the associated stripe id by looking up the corresponding registration_payment.

I think this would also allow you to get rid of the `payment.refunded_at` field, instead you'd look up the corresponding `registration_payment` and look at its `created_at` field. Also, I suspect that a `refunded_at` field doesn't make a lot of sense if you can partially refund a payment?